### ISSUE TITLE
New article: Synchronous and asynchronous communication tools

### ISSUE THREAD

**Issue by jwflory** [unicef/inventory#113]
_This came up in conversation with BX Smart Labs in their monthly Open Source Mentorship check-in call on 6 January 2022_.

## Metadata

* **Article/page title**: Synchronous and asynchronous communication tools
* **Category**: Project Management
* **Tags**: `communication`, `community`

## Content description

* **Intended audience(s)**: Open source community leads, developers, project managers, and key collaborators in on-going collaboration with a project
* **Why should this page exist?**: To explain a key difference between virtual and online communication platforms, provide examples of each, and offer pros/cons for each type of comms tool.

_Content description_:

1. Intro to chat platforms for open source communities
2. What are synchronous communication tools? (pros/cons)
3. What are asynchronous communication tools? (pros/cons)
4. Picking a tool for your community
5. Case studies and examples

**Comment by Syed-Ansar** [comment]
can @jwflory explain me what to do ?

**Comment by jwflory** [comment]
Hi @Syed-Ansar, thank you for your interest on the UNICEF Open Source Inventory! I already assigned this issue to me because it is an article I wanted to write with a specific vision in mind. It comes from feedback I have working with [UNICEF Innovation Fund start-up companies](https://www.unicefinnovationfund.org). It is better to find other things to work on.

Are you more interested in writing content, or working on code and UI/UX? If you are more interested in the coding side, you may want to check out the repo for the Hugo theme used to power this website instead.

https://github.com/unicef/inventory-hugo-theme

**Comment by jwflory** [comment]
Also, as a note, I am out of office this week so I will be slow to reply. Next week, I will be back online and can better respond to queries for help.

**Comment by Syed-Ansar** [comment]
Yes! I am interested in coding. Yesterday I tried to setup repository on my local computer but I was not able to do correctly. So would be happy if I get any help from your side. Thank you @jwflory

**Comment by jwflory** [comment]
@Syed-Ansar I think this intern on-boarding guide will be useful until I am back in the office and can troubleshoot more questions:

https://unicef.github.io/inventory/meta/inventory/intern-guide/
